src/Microsoft.ML.Core/Microsoft.ML.Core.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.ComponentModel.Composition" Version="4.5.0" />
```

Comment:
You'll need to add this to the nupkg dependencies.

[https://github.com/dotnet/machinelearning/blob/a55b2a6ba96c6caba414ecb6d229e81509cce581/pkg/Microsoft.ML/Microsoft.ML.nupkgproj#L11-L16](https://github.com/dotnet/machinelearning/blob/a55b2a6ba96c6caba414ecb6d229e81509cce581/pkg/Microsoft.ML/Microsoft.ML.nupkgproj#L11-L16)